skimage/__init__.py

```diff
@@ -98,14 +101,19 @@ def _test(doctest=False, verbose=False):
             return 1
 
 
+# register the viridis colormap
+register_cmap('viridis', viridis)
```

Comment:
Regarding the actual PR... registering the viridis colormap into matplotlib might be problematic, since matplotlib will also ship with it by default soon. Depending on how `register_cmap` works, this might either raise an error (attempting to redefine an existing colormap), or else mean that importing skimage could change how matplotlib plots look by default (e.g. due to tiny differences from using a different linearization of the colormap), which could break tests based on comparing pixel values.

Alternatives: (a) don't try registering the name `"viridis"`, just use
```
default_colormap = viridis   # note the lack of quotes
```
(b) check whether matplotlib has a viridis built-in; if so use that instead of defining our own.